snap/snapcraft.yaml

```diff
@@ -163,7 +162,6 @@ parts:
       - LD_LIBRARY_PATH: $CRAFT_STAGE/usr/lib/$CRAFT_ARCH_TRIPLET:/usr/lib/$CRAFT_ARCH_TRIPLET${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}
     override-pull: |
       craftctl default
-      git apply $CRAFT_PROJECT_DIR/patches/fix-mouse.diff
```

Comment:
if you delete this line, there is no reason to keep the `override-pull:` block at all, would you mind deleting the whole block (i.e. the two lines before the git apply)